start.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+source `dirname "${BASH_SOURCE[0]}"`/.util.sh
+
+source `dirname "${BASH_SOURCE[0]}"`/.configure.sh
+
+source `dirname "${BASH_SOURCE[0]}"`/clusters/${CLUSTER}/start.sh
+source `dirname "${BASH_SOURCE[0]}"`/registries/${REGISTRY}/start.sh
```

Comment:
same for line 7 & 8, what about `./` instead?

Reply:
They need to be part of the same shell to share functions